concrete_datastore/api/v1/filters.py

```diff
@@ -468,9 +502,9 @@ def get_custom_filters(param):
             if target_field not in filterset_fields:
                 return None
 
-            target_field_type = queryset.model._meta.get_field(
-                target_field
-            ).get_internal_type()
+            target_field_type = get_filter_field_type(
+                queryset.model, target_field
+            )
             if target_field_type not in RANGEABLE_TYPES:
```

Comment:
Did you ensure `RANGEABLE_TYPES` is a list of `str` ?

Reply:
No but it's define above as this:
```python
RANGEABLE_TYPES = (
    'DateTimeField',
    'DateField',
    'DecimalField',
    'IntegerField',
    'FloatField',
)
```